omega/omega-context/src/main/java/org/apache/servicecomb/pack/omega/context/OmegaContext.java

```diff
@@ -72,4 +72,10 @@ public String toString() {
         ", localTxId=" + localTxId.get() +
         '}';
   }
+
+  public void verify(){
+    if(this.globalTxId == null){
+      throw new RuntimeException("OmegaContext globalTxId is empty");
```

Comment:
We could add more information about the error, such as "Please check if you setup the pack transport handler rightly."